kernel_gateway/gatewayapp.py

```diff
@@ -277,6 +279,27 @@ def keyfile_default(self):
     def client_ca_default(self):
         return os.getenv(self.client_ca_env)
 
+    kernel_spec_manager = Instance(KernelSpecManager, allow_none=True)
+
+    kernel_spec_manager_class = Type(
+        default_value=KernelSpecManager,
+        config=True,
+        help="""
+        The kernel spec manager class to use. Should be a subclass
+        of `jupyter_client.kernelspec.KernelSpecManager`.
+
+        The Api of KernelSpecManager is provisional and might change
```

Comment:
We shouldn't have API warnings from Jupyter Client in the kernel gateway code, I think.

Reply:
I wasn't sure what the format for help would be so I copied it from [Jupyter Notebook](https://github.com/jupyter/notebook/blob/master/notebook/notebookapp.py#L997-L1007)